docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/README.md

```diff
@@ -0,0 +1,121 @@
+# {{ cookiecutter.app_verbose_name }}
+
+## Preparing for Docker
+
+Before building the Docker container, run
+```bash
+mv env.example .env
+```
+
+## Developing
+
+Development requires a local installation of [Docker](https://docs.docker.com/install/)
+and [Docker Compose](https://docs.docker.com/compose/install/).
+
+Build application containers:
+
+```bash
+docker-compose up --build
+```
```

Comment:
Docker Compose will auto build for you, if the image hasn't been built before. So I think we can axe this and start with loading the initial content.